src/main/java/core/basesyntax/report/ReportBuilderImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.report;
+
+import core.basesyntax.storage.FruitStorage;
+
+public class ReportBuilderImpl implements ReportBuilder {
+    @Override
+    public String buildReport(FruitStorage storage) {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity\n");
+        storage.getAllFruits().forEach((fruit, quantity) -> {
+            report.append(fruit).append(",").append(quantity).append("\n");
```

Comment:
write in stack

Reply:
mentor meant one appent at a line, stack as in "stacked on top of each other" :)

Reply:
> mentor meant one append at a line, stack as in "stacked on top of each other" :)

got it, thanks ;D